lib/aac/utils.js

```diff
@@ -24,13 +24,35 @@ var ADTS_SAMPLING_FREQUENCIES = [
   7350
 ];
 
+var parseId3TagSize = function(header, byteIndex) {
```

Comment:
Code is unchanged. I just moved it above `isLikelyAacData` as we use it there now.